cloudbuild/main.tf

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+resource "google_cloudbuild_trigger" "default-push-trigger" {
+    provider = google-beta  # Use Beta version for GitHub support
+    project = var.gcp_project_id
+    github {
+        owner = var.github_owner
+        name = var.github_repo_name
+        push {
+            branch = ".*"
```

Comment:
could be useful to variable-ize branch.

Reply:
For example if in a contributor's workflow is on their fork they rage commit / push to a wip branch and then merge to master on their fork when they are ready run CI checks.

Reply:
bump. thoughts on variablizing this branch pattern?

Reply:
done in 1344d825e0fc467713a3c1e69d5297685e304e30